docs/conf.py

```diff
@@ -84,7 +84,7 @@ def setup_clang():
         return
 
     try:
-        cindex.Index.create()
+        pass  # cindex.Index.create()
```

Comment:
@antocuni Are you okay with me just deleting this call `cindex.Index.create()` now? It segfaults consistently on my machine. Not sure if your setup still needs it or not though.

Reply:
it segfaults also for me, I'm fine with deleting.
But maybe keep a comment saying that on ubuntu you need `apt install libclang-10-dev`?